app/channel.go

```diff
@@ -1330,13 +1325,11 @@ func (a *App) addUserToChannel(user *model.User, channel *model.Channel, teamMem
 
 	newMember, nErr = a.Srv().Store.Channel().SaveMember(newMember)
 	if nErr != nil {
-		mlog.Error("Failed to add member", mlog.String("user_id", user.Id), mlog.String("channel_id", channel.Id), mlog.Err(nErr))
```

Comment:
We might be losing a bit of info here. Maybe we can still include it in the params map for the appError.

Reply:
yeah, we did. We should add the Err(nErr) back

Reply:
Or will we not get traceback on INFO level logging?

Reply:
You should still get the main error given it's a `StatusInternalServerError`. What will be missing is the inner error (from the store/database layer essentially).

Reply:
yeah, the traceback, seems that is useful. As in the non-updated version we have deployed the same error occured but we were able to find out it was because of pk constraint on trying to add the user again to the channel.

Reply:
Thanks for the feedback @source-punk . I agree that such errors should be surfaced. Noting this down to get fixed.

Reply:
Thanks! 🙏

Reply:
Fixed in https://github.com/mattermost/mattermost-server/commit/baf8eda69957864366f1c6120b0647d079f739ea

Reply:
Thanks!! :)